inputfiles/idl/SVG - Basic Data Types and Interfaces.widl

```diff
@@ -227,3 +227,9 @@ interface mixin SVGZoomAndPan {
 interface mixin SVGURIReference {
   [SameObject] readonly attribute SVGAnimatedString href;
 };
+
+[Exposed=Window]
+interface SVGCursorElement : SVGElement {
+  readonly attribute SVGAnimatedLength x;
+  readonly attribute SVGAnimatedLength y;
+};
```

Comment:
Thanks for jumping in.
Is this a change in the widl from upstream? 
I suspect no, so the [correct place](https://github.com/Microsoft/TSJS-lib-generator#input-files) would be the addTypes.json